mm2src/mm2_main/src/rpc.rs

```diff
@@ -314,23 +282,19 @@ async fn rpc_service(req: Request<Body>, ctx_h: u32, client: SocketAddr) -> Resp
 
     let res = try_sf!(process_rpc_request(ctx, req, req_json, client).await, ACCESS_CONTROL_ALLOW_ORIGIN => rpc_cors);
     let (mut parts, body) = res.into_parts();
-    let body_escaped = {
-        let body_utf8 = match std::str::from_utf8(&body) {
-            Ok(body_utf8) => body_utf8,
-            Err(_) => {
-                return Response::builder()
-                    .status(500)
-                    .header(ACCESS_CONTROL_ALLOW_ORIGIN, rpc_cors)
-                    .header(CONTENT_TYPE, APPLICATION_JSON)
-                    .body(Body::from(err_to_rpc_json_string("Non UTF-8 output")))
-                    .unwrap();
-            },
-        };
-        let escaped = escape_answer(body_utf8);
-        escaped.as_bytes().to_vec()
+    let body_bytes = match std::str::from_utf8(&body) {
+        Ok(body_utf8) => body_utf8.as_bytes().to_vec(),
+        Err(_) => {
+            return Response::builder()
+                .status(500)
+                .header(ACCESS_CONTROL_ALLOW_ORIGIN, rpc_cors)
+                .header(CONTENT_TYPE, APPLICATION_JSON)
+                .body(Body::from(err_to_rpc_json_string("Non UTF-8 output")))
+                .unwrap();
+        },
```

Comment:
Do we really need to check if the bytes could be `utf-8`ed at this point?
We should trust us.